CONFIGURE.sh

```diff
@@ -19,35 +36,33 @@ HSS_REPOSITORY=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 # assume the TA grading repo is in the same location
 # NOTE: eventually the TA grading repo will be merged into the main repo
-#TAGRADING_REPOSITORY=$HSS_REPOSITORY/../GIT_CHECKOUT_TAgrading
 TAGRADING_REPOSITORY=$HSS_REPOSITORY/TAGradingServer
```

Comment:
It's on line 118 of the file still. Do you want to replace that reference with $HSS_REPOSITORY/TAGradingServer?